ffi/Cargo.toml

```
@@ -25,7 +25,7 @@ whoami = "1.4"
 sspi = { path = "..", features = ["network_client", "dns_resolver"] }
 ffi-types = { path = "../crates/ffi-types", features = ["winscard"], optional = true }
 picky-asn1-der = "0.4"
-
+uuid = { version = "1.4.1", default-features = false }
```

nitpick: only specify the minor number
```suggestion
uuid = { version = "1.4", default-features = false }
```

---

done